internal/gpsd/gpsd.go

```diff
@@ -50,6 +52,12 @@ type Conn struct {
 	closed       bool
 }
 
+type GPSdClient interface {
+	Dial(addr string) (GPSdClient, error)
+	Watch(enable bool)
+	Next() (*TPV, error)
+}
+
```

Comment:
This is unused? If so, please omit it 😊

Reply:
I am looking at storing the Locator in the user.json for the multi-user reloader.  Like KM4ACK's user swap by config swap
in the PAT Menu 12.  GPSd is great and I have one pi feeding 4 pi's locator.  only have a 20 minute gap every other month.
in mailbox/CallSign/user.json is (CALLSIGN, Name, TAC), Locator (EL29ER27),  WinLink/user (Password), Roll (Owner, Radio Op, Non-Ham), On Duty(00-23), NTS (routing itu-r2, NOAM, USA, TX, STX,)  HW ACCESS (Telnet, Wifi2, Wifi5, Ardop, ### Ax25, VaraHF, VaraFM, VaraSat)
Just have to get the Variables loaded at the right time for each Network http connection session or postoffice push/pull.
This lets windows and mac users to get multi-user.... Roll->Multi-user->PostOffice->NTS
Herbert KD5PQJ